wasm/js/renderer.js

```diff
@@ -717,5 +717,7 @@ Rive.onRuntimeInitialized = function () {
             _animationCallbackHandler.requestAnimationFrame.bind(_animationCallbackHandler);
     Rive['cancelAnimationFrame'] =
             _animationCallbackHandler.cancelAnimationFrame.bind(_animationCallbackHandler);
+    Rive['enableFPSCounter'] =
```

Comment:
it may be frowned upon...but in this case, I wonder if we might want to put this on the `window` object. I'm thinking in cases where for the rive site for example, we have multiple high-level RIve objects existing at the same time. Because this is more of a global tool - we could set something like `enableRiveFPSCounter()` and then a client with multiple of these Rive objects doesn't need to reason with which one to set the API to `true` here. wdyt? also @avivian

Reply:
I don't think that would work because this can only measure fps for "rive.requestAnimationFrame" calls on the given rive object.

Would it be possible for us to only have a single high-level rive object? What's the reason for having multiple? Having one is definitely better for offscreen webgl performance.

Reply:
Every high-level Rive object is tied to a specific rive file. This is how the parcel example works too. So if a user wants to lay out 5 animations from 5 different files, we'll have 5 render loops going on, and 5 fps times to report. We can certainly display fps per rive animation